From: Florian Preinstorfer <nblock@archlinux.us>
Date: Tue, 17 May 2016 19:41:57 +0200
Subject: Do not use lib-prefix on Unix

---
 src/src.pro | 6 +-----
 1 file changed, 1 insertion(+), 5 deletions(-)

diff --git a/src/src.pro b/src/src.pro
index 872676d..67a8d52 100644
--- a/src/src.pro
+++ b/src/src.pro
@@ -151,8 +151,4 @@ macx {
 
 LIBS += -lcryptopp
 LIBS += -lz
-
-CONFIG += link_pkgconfig
-unix:PKGCONFIG += libpcsclite
-macx:PKGCONFIG += libpcsclite
-win32:LIBS += libwinscard
+LIBS += -lpcsclite
